contributing-guides/style-guide.md

```diff
@@ -47,32 +47,46 @@ in order to allow `tldr` clients to highlight them.
 
 Keep the following guidelines in mind when choosing tokens:
 
-1. Use short but descriptive tokens,
-   ex. `{{source_file}}` or `{{wallet.txt}}`.
-2. Use [`snake_case`](https://en.wikipedia.org/wiki/Snake_case) for multi-word tokens.
-3. Use `{{filename}}` rather than `{{file_name}}`.
-4. For any reference to paths to files or directories, use the format `{{path/to/<placeholder>}}`.
-   For example, `ln -s {{path/to/file}} {{path/to/symlink}}`.
-   In case of a possible reference both to a file or a directory, use `{{path/to/file_or_directory}}`
-5. Follow the `{{path/to/<placeholder>}}` convention for all path-related commands, except when the
-   file location is implicit.
-6. If a command expects the file to have a particular extension, use it.
-   For example, `unrar x {{compressed.rar}}`.
-   In case a generic extension is needed, use `{{.ext}}`, but **only** if an extension is required.
-   For instance, in find.md's example "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
-   using `{{*.ext}}` explains the command without being unnecessarily specific;
-   But in a command like `wc -l {{file}}`, using `{{file}}` (without extension) is sufficient.
-7. If the example is clearer with an actual value rather than a generic placeholder, use the actual value.
-   For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
-8. If a command performs irreversible changes to a file system or to user's devices, then write every example in a way that they cannot be unmindfully copy-pasted by the user.
-   For example, instead of `ddrescue --force --no-scrape /dev/sda /dev/sdb` write `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}` and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
+### Naming
+- Use short but descriptive tokens,
+  such as `{{source_file}}` or `{{wallet.txt}}`.
+- Use [`snake_case`](https://wikipedia.org/wiki/snake_case) for multi-word tokens.
+- Use an actual value rather than a generic placeholder where appropriate.
+  For example, use `iostat {{2}}` rather than `iostat {{interval_in_secs}}`.
+
+### Paths
+- Use `{{filename}}` rather than `{{file_name}}`.
+- For any reference to paths to files or directories,
+  use the format `{{path/to/<placeholder>}}`,
+  except when the file location is implicit.
+  For example, `ln -s {{path/to/file}} {{path/to/symlink}}`.
```

Comment:
```suggestion
  except when the file location is implicit.
  When the path cannot be relative, but has to start at the root of the filesystem, use `{{/path/to/<placeholder>}}`
  For example, `ln -s {{/path/to/file_or_directory}} {{path/to/symlink}}`.
```
The `ln` page has been updated

Reply:
I disagree with that update to the `ln` page though. You can perfectly well provide it a relative path. Actually I can't think of a single example where only absolute paths are okay, doesn't make much sense to me tbh.

Reply:
While it is possible, I wouldn't recommend it. (I explained why in #6087) But we can still use another command, like `get {{/path/remote_file}}`

Reply:
Well, I prefer relative symlinks most of the time, so I think another example would indeed make more sense. But this `get` has no TLDR page, and since it is a quite generic term I am unsure what program you are referring to?

Reply:
I think it's more about normalization of all pages. If some pages use relative paths while some pages use absolute path, it would look chaotic. So the current practice in tldr is using absolute path as many as possible.

Reply:
> Well, I prefer relative symlinks most of the time, so I think another example would indeed make more sense. But this `get` has no TLDR page, and since it is a quite generic term I am unsure what program you are referring to?

My bad, I was talking about the `sftp` page.

@blueskyson what do you mean? We always use relative paths when possible. This part here is just about how to write absolute paths for commands which need it. Some commands can't use relative paths.

Reply:
I mean when to use of `{{path/to/file}}` or `{{file}}`.

Reply:
In this guide we advice to use `{{path/to/file}}` instead of `{{file}}`. In case the command get's too long (which isn't the case very often), we can still change it in the PR.